python_as3.py

```diff
@@ -0,0 +1,76 @@
+class Calculator:
+
+    def __init__(self,num_1,num_2): #using __init__ to initialize the variables
+        self.num_1=num_1
+        self.num_2=num_2
+
+
```

Comment:
inside a class, only one line space is enough. outside the class you need to have 2 lines of space.

Reply:
Ok sir. I had made the changes